src/xenia/kernel/xboxkrnl/xboxkrnl_io.cc

```diff
@@ -105,6 +105,11 @@ dword_result_t NtCreateFile(lpdword_t handle_out, dword_t desired_access,
     return X_STATUS_OBJECT_NAME_INVALID;
   }
 
+  if (target_path.back() == '\\' &&
```

Comment:
Not really sure overall about this way of checking whether it's a directory, though need to read how that object structure is filled, and where does the path originate from (if from the game, it will not necessarily probably include the \, if from Xenia's file system code, then maybe). Also if the path is empty, this will cause an error.